cypress/integration/dashboard-gitlab-pipeline.spec.js

```diff
@@ -4,7 +4,7 @@ context('Gitlab pipeline', () => {
     });
 
     it('pushes a successful GitLab pipeline', () => {
-        for (let i = 1; i <= 27; i++) {
+        for (let i = 9; i <= 27; i++) {
```

Comment:
I would still run the first 8 but with a delay of 100ms, so we know if something breaks here. e.g. if they are not ignored properly anymore. This only costs less than a second instead of waiting the full 8 seconds everytime.

Reply:
Good point, will make sure that gets changed soon